packages/odyssey-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -79,12 +86,15 @@ const Checkbox = forwardRef<HTMLInputElement, Props>((props, ref) => {
     internalRef.current.indeterminate = indeterminate;
   }, [indeterminate, internalRef]);
 
+  const ariaDescribedBy = useCx(error && `${oid}-error`);
```

Comment:
see my previous comments about the implicit API this introduces by using this `-error` or `-hint` suffix. Lets fix this in the immediate follow up PR